.github/workflows/installer.yml

```diff
@@ -0,0 +1,57 @@
+name: Installer integration
```

Comment:
I think we should avoid adding too many workflows, rather than adding jobs to an existing workflow, because the workflows show up completely separately in the UI. This is why the monorepo uses two very large workflow files, and `files_changed` to detect whether to skip jobs.

Reply:
> because the workflows show up completely separately in the UI

What's the downside of that? 

> This is why the monorepo uses two very large workflow files

Isn't it easy to read/maintain if you have separate workflow files? Then they're isolated from each other and you're less likely to break something if you modify something in the workflow.

Reply:
It's a matter of personal preference, and your call to make of course. I just find this:

![image](https://user-images.githubusercontent.com/52710/124990927-dcd72180-dff5-11eb-93b5-b050db5f9880.png)

to make it more difficult to tell which of the tests passed/failed, than this view:

![image](https://user-images.githubusercontent.com/52710/124991049-01cb9480-dff6-11eb-9b40-e6dd78b93e3a.png)

GitHub Actions also supports inter-job dependencies but it's harder to do inter-workflow dependencies. It could be that the tradeoffs we were making are different from the tradeoffs you're making, of course, so you'll have to do what makes more sense for you :grin:

Reply:
Ah, I see what you mean now. Thanks for the screenshots! Yeah, I can see why that might be easier. I guess we'll leave it to @code-asher then to make the call!

Reply:
My first impression was that I liked them split up weirdly enough. But it's definitely possible we may find that to be unwieldy so I think we could see how this goes and then if ends up being frustrating we can combine them.